satellite/gracefulexit/endpoint.go

```diff
@@ -344,6 +352,14 @@ func (endpoint *Endpoint) doProcess(stream processStream) (err error) {
 		if !morePiecesFlag && pendingCount == 0 {
 			processMu.Unlock()
 
+			isDisqualified, err := endpoint.handleDisqualifiedNode(ctx, nodeID)
```

Comment:
Do we need to query this for every step? I think calling it once before doing this should be sufficient.

Reply:
This would only be called once a node has finished transfer all the pieces and is about to finish graceful exit